src/Eris/Generator/AssociativeArrayGenerator.php

```diff
@@ -14,7 +14,12 @@ function associative(array $generators)
 class AssociativeArrayGenerator implements Generator
 {
     private $generators;
-    
+
+    /**
+     * @var TupleGenerator
+     */
```

Comment:
@giorgiosironi 

Let me update!

Happy to make `php-cs-fixer` a development dependency, too!